appl/test/auditdns.c

```diff
@@ -66,6 +68,24 @@ gethostbyname2(const char *name, int af)
 
 #endif	/* HAVE_GETHOSTBYNAME2 */
 
+struct hostent *
+gethostbyaddr(const void *addr, socklen_t len, int af)
+{
+    const socklen_t maxlen[] = {
+	[AF_INET] = sizeof(struct in_addr),
```

Comment:
This won't work for Windows.  Did I miss anything similar earlier?  Anyways, the reason is that we still support older MSFT tools that don't fully support C99.